framework/resources/Datasources/parquet_storage/filter/pushdown/varchar_decimal/create_hive_tables.ddl

```diff
@@ -0,0 +1,41 @@
+create database if not exists filter_pushdown;
+
+drop table if exists filter_pushdown.customer;
+create external table filter_pushdown.customer(
+  c_custkey int,
+  c_name string,
+  c_address string,
+  c_nationkey int,
+  c_phone string,
+  c_acctbal decimal(6,2),
+  c_mktsegment string,
+  c_comment string
+)
+stored as parquet
+location 'dfs_location/customer';
+
+drop table if exists filter_pushdown.part_old;
+create external table filter_pushdown.part_old(
+  p_partkey int,
+  p_name string,
+  p_mfgr string,
+  p_brand string,
+  p_type string,
+  p_size int,
+  p_container string,
+  p_retailprice bigint,
+  p_comment string
+)
+stored as parquet
+location 'dfs_location/part_old';
```

Comment:
Not sure what's with the "_old" name..

Reply:
I'm guessing it's got to do with the 1.14.0 metadata file.. any reason we have two versions? Any changes in behavior / new features? And did we find any issues while testing old vs new- if not we could just get rid of the older version and just have the new one.

Reply:
Old means generated by Drill 1.14.0. There were changes in statistics since then. Wanted to check that old statistics is handled adequately.

Reply:
Instead of old, can you probably use sub-dirs to indicate the Drill version more clearly? So in the future if we make changes in v1.18.0, we can have 1.14.0, 1.16.0 and 1.18.0 then.

Reply:
Sounds reasonable. Done.